cluster/etcd_service_discovery.go

```diff
@@ -339,9 +340,11 @@ func (sd *etcdServiceDiscovery) InitETCDClient() error {
 	logger.Log.Infof("Initializing ETCD client")
 	var cli *clientv3.Client
 	var err error
+	etcdClientLogger, _ := logutil.CreateDefaultZapLogger(logutil.ConvertToZapLevel("error"))
```

Comment:
because this doesn't use our internal logger. This is to tap into the etcd client library's own logger and set it to a different log level than our internal logger (we don't want the noisy warn+info messages from the etcd library, but we do want the warn/info messages which are controlled by the log level settings on our app).

Reply:
Does ETCD support replacing its internal logger with an instance of ours? Like implementing an interface

Reply:
It doesn't and even if it did, the noisy warning message we want to get rid of is also not something which we can set to debug. So we'll need to have two loggers with different levels (the app one that the user can control, and the etcd one, set to error level explicitly)